content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

```diff
@@ -91,6 +91,8 @@ In addition to various view options, the :guilabel:`Requests for Quotation` dash
 .. seealso::
    :doc:`../../../essentials/search`
 
+.. _inventory/create-new-rfq:
```

Comment:
Please correct app name and maintain the convention used throughout doc :)

Be sure to update the associated ref(s)

```suggestion
.. _purchase/manage_deals/create-new-rfq:
```